README.md

```diff
@@ -21,6 +21,11 @@ The version of kibana to install (major and minor only).
 
 The FQDN or IP address and port Kibana should use.
 
+    kibana_config_dir: "/etc/kibana"
```

Comment:
Could you just set this to `kibana_config_dir: /opt/kibana/config`... and then drop all the extra tasks that are checking a bunch of different directories?

Out of the box, it would work with the current setup for this role, and if you need to change it, you can figure that out and override the variable one time.

Reply:
No, I won't. See https://github.com/geerlingguy/ansible-role-kibana/issues/13 .
The problem is that the configuration directory defaults to /etc/kibana in at least current versions for Ubuntu/CentOS/RHEL , but in your setup it is /opt/kibana/config.
This is the reason why I try to guess the correct directory.
Perhaps this guessing needs further improving. But it is definitely not a good idea to stick to /opt/kibana/config .
Alternatively, I think it would be possible to select config directory based on detected distribution/environment.